src/v2/es-utils.js

```diff
@@ -181,24 +194,37 @@ class ESClass {
     } catch (err) {
       if (err) index = '.kibana';
     }
-    if (typeof index === 'object') index = Object.keys(index)[0];
+    if (typeof index === 'object') index = (parseInt(this.env.ES_MAJOR, 10) > 6)
+      ? Object.keys(index.body)[0]
+      : Object.keys(index)[0];
+
     const exists = await this.client
       .indices
       .exists({ index: index });
-    if (exists) {
+    if (exists || exists.body) {
```

Comment:
Is there a case where `exists` can be falsy, while `exists.body` is truthy? The second condition seems redundant